regulations/static/regulations/css/less/module/comment-review.less

```diff
@@ -244,6 +244,10 @@ comment-review.less contains styles for Review Your Comment page
       list-style-type: disc;
     }
 
+    .disclaimer-text {
+      display: none;
+    }
```

Comment:
Rather than hiding the text in CSS (which makes it inaccessible to non JS people), can we hide it within the JS?